src/frontend/components/Select.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import './style.scss'
+const Select =(props)=> {
+   return (
+         <select name ={props.name} className="blinkas-black" onChange={props.onChange}>
+           {
+             props.options.map((item , index)=>(
+               <option key ={index} value = {item}>{item}</option>
+             ))
+           }
+         </select> 
+      
```

Comment:
Fix all spacing, indentation and newline. Most of them are not needed. You should consider setting up `prettier` on your VS code editor to help with this.